.github/workflows/tests.yaml

```diff
@@ -47,11 +45,19 @@ jobs:
         python3.10 -m pip install tox
         rm get-pip.py
     
-    - uses: balchua/microk8s-actions@v0.3.2
-      with:
-        channel: '1.28/stable'
-        addons: '["hostpath-storage"]'
+    - name: Install dependencies
+      run: |
+        # Removing docker as it is blocking canonical k8s bootstrap
```

Comment:
Should we link to here even if we 're not using their workaround? https://documentation.ubuntu.com/canonical-kubernetes/latest/snap/howto/install/dev-env/#containerd-conflicts